var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -32,7 +32,7 @@ class Seacas(CMakePackage):
     # ###################### Versions ##########################
     version("master", branch="master")
     version(
-        "2023-10-24", sha256="f93bf0327329c302ed3feb6adf2e3968f01ec325084a457b2c2dbbf6c4f751a2"
+        "2023-11-27", sha256="fea1c0a6959d46af7478c9c16aac64e76c6dc358da38e2fe8793c15c1cffa8fc"
```

Comment:
In case someone had the misfortune to install that version, just add the new one and mark the old with `deprecated=True`.

Reply:
done